### PR TITLE
Closed #17 Final

### LINKED ISSUE #17: Connie Gale

### PR DESCRIPTION
**Pull request by DarkerLonging**

### CODE REVIEW

File: cgale/art.html [hunk 1]
```diff
@@ -177,6 +177,7 @@ <h1 class="page-header">Connie Gale Art
             <div class="row">
                 <div class="col-lg-12">
                     <p>Copyright &copy; Connie Gale </p>
+                      <p> Coded By AJ Donald </p>
```
Comment (gemohandy) [inline cgale/art.html]: This single line is causing a conflict.

File: cgale/art2.html [hunk 2]
```diff
@@ -155,6 +155,7 @@ <h1 class="page-header">Connie Gale Art
             <div class="row">
                 <div class="col-lg-12">
                     <p>Copyright &copy; Connie Gale </p>
+                      <p> Coded By AJ Donald </p>
```
Comment (gemohandy) [inline cgale/art2.html]: It keeps showing up in all the Connie Gale pages

File: cgale/index.html [hunk 3]
```diff
@@ -116,6 +116,7 @@
             <div class="row">
                 <div class="col-lg-12">
                     <p>Copyright &copy; Connie Gale 2016</p>
+                    <p> Coded By AJ Donald </p>
```
Comment (gemohandy) [inline cgale/index.html]: Really odd, as this error has absolutely nothing to do with Connie Gale.

File: cgale/info.html [hunk 4]
```diff
@@ -104,6 +104,7 @@ <h1 class="tagline">Information about Connie</h1>
             <div class="row">
                 <div class="col-lg-12">
                     <p>Copyright &copy;Connie Gale</p>
+                      <p> Coded By AJ Donald </p>
```
Comment (gemohandy) [inline cgale/info.html]: Also, is DarkerLonging AJ Donald?

File: cgale/linemen.html [hunk 5]
```diff
@@ -138,6 +138,7 @@ <h1 class="page-header">Linemen art
             <div class="row">
                 <div class="col-lg-12">
                     <p>Copyright &copy; Connie Gale 2016</p>
+                      <p> Coded By AJ Donald </p>
```
Comment (gemohandy) [inline cgale/linemen.html]: I don't know what to put here. I'm kind of out of things to say about this reoccuring line.

File: emmalake/artists.html [hunk 6]
```diff
@@ -15,55 +15,62 @@
 <a class="a" href="gallery.html">Gallery</a>
 </div>
 <div id="content">
-
+<ul>
```
Comment (gemohandy) [inline emmalake/artists.html]: The ^M comes from operating system conflicts.

File: emmalake/index.html [hunk 7]
```diff
@@ -17,12 +17,24 @@
...
-<p>Some 80 artists and critics have served as workshop leaders over the 4 decades that workshops were held, the inaugural session starting with Vancouver artist Jack SHADBOLT. Some of the more notable leaders included Kenneth Noland (1963), Jules Olitski (1964) and Anthony Caro (1977), but probably the most well-known were Barnett Newman (1959) and Clement GREENBERG(1962). Their association with American post-war modernist art has possibly made these individuals responsible for giving the Emma Lake workshops a modernist focus. To attribute the entrenchment of modernist art on the prairies to the influence of Emma Lake and its leaders, however, discounts the importance of the effects of geography and the particularily cooperative character of the prairie, particularily Saskatoon, art scene.</p>
+<p>The Canadian Encyclopedia describes Emma Lake as a “remote location in northern Saskatchewan.”
```
Comment (gemohandy) [inline emmalake/index.html]: I don't know why this file is causing conflicts.

File: emmalake/artists.html [hunk 8]
```diff
@@ -51,23 +51,23 @@
...
 <br>
-<li><a class="a" href=".html">1976  Toni Onley, Douglas Haynes, Hubert Hohn, Sidney Tillim, Harold Feist, Hans Dommasch</a></li>
+<li>1976  Toni Onley, Douglas Haynes, Hubert Hohn, Sidney Tillim, Harold Feist, Hans Dommasch</li>
 <br>
 <li>Dan Solomon, Andre Fauteux, Otto Rogers, Bruce O'Neil, Joseph Reeder, and Alan Gliko (guest leaders)</li>
 <br>
 <li><a class="a" href="anthonycaro.html">1977 Anthony Caro (workshop leader)</a></li>
 <br>
-<li>1978 No Workshop </li>
+<li><a href="../saskartists.ca">1978 No Workshop</a></li>
```
Comment (BmanDesignsCanada) [inline emmalake/artists.html]: These should not head back to the homepage. Leave them as no links.

File: emmalake/artists.html [hunk 9]
```diff
@@ -7,7 +7,7 @@
 <body>
 <div id="wrapper">
 <div id="header">
-<h1> Emma Lake Artists' Workshops </h1>
+<h1><a href="../saskartists.ca"> Emma Lake Artists' Workshops</a></h1>
```
Comment (BmanDesignsCanada) [inline emmalake/artists.html]: This should also not be a link

File: emmalake/artists.html [hunk 10]
```diff
@@ -51,23 +51,23 @@
 <br>
 <li>1970 Ronald B. Kitaj (Workshop Leader)</li>
 <br>
-<li>1971 No Workshop</li>
+<li><a href="../saskartists.ca">1971 No Workshop</a></li>
```
Comment (BmanDesignsCanada) [inline emmalake/artists.html]: This should be left as no link.

**Comment by BmanDesignsCanada** [comment]
Redo these changes on a fork of master, this is too badly broken.

**Pull request closed without merging**